collections/nemo_asr/nemo_asr/helpers.py

```diff
@@ -134,32 +138,41 @@ def process_evaluation_batch(tensors: dict, global_vars: dict, labels: list):
                                                        labels=labels)
 
 
-def process_evaluation_epoch(global_vars: dict, tag=None, logger=None):
+def process_evaluation_epoch(global_vars: dict,
+                             use_wer=True, tag=None, logger=None):
```

Comment:
From style guide: `If a method has arguments that don't fit into one line, each argument should be in its own line for readability.`